### PR TITLE
Menu icon

### PR DESCRIPTION
**Pull request by Sly-Cloud**
- Made the hamburger visible
- Added Script to make it appear when clicked

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -255,66 +90,249 @@ <h2>
...
+          <li class="nav-link-item">
+              <a href="#" class="nav-link" id="portfolio">Portfolio</a>
+          </li>
+          <li class="nav-link-item">
+              <a href="#" class="nav-link" id="about">About</a>
+          </li>
+          <li class="nav-link-item">
+            <a href="#" class="nav-link" id="contact">Contact</a>
+          </li>
+        </ul>
```
Comment (sinansevgi) [inline index.html]: [OPTIONAL]
- [ ] You should put proper links to your menu items.

File: style.css [hunk 2]
```diff
@@ -37,20 +41,100 @@ hr {
...
+.hamburger-menu > div:last-child {
+  margin-bottom: 0;
+}
+
+.nav-list {
+  display: none;
 }
 
-.status-bar {
+.menuToggle {
```
Comment (sinansevgi) [inline style.css:115]: - [ ] Mobile menu doesn't have a proper background blur as seen in the design. This makes the menu hard to read. Please put proper background blur to your mobile menu. You can use [ filter CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) for this

Your version
Screenshot 2021-06-21 at 21-01-18 Portfolio Sneak Peak

Orginal Design
Screenshot 2021-06-21 at 21-01-37 Figma

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Sly-Cloud and @olawale-o, 

Great Work! 
I can see you configured linters correctly and it doesn't show any errors :+1:  
The mobile menu functionality works as expected (:+1: ) but links are missing. 
There is a small styling issue that you can easily fix and continue to the next step. 


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: - Not addressed

Comment (rloterh) [reply index.html]: > [OPTIONAL]
> 
> * [ ]  You should put proper links to your menu items.

Kindly endeavor to work on this recommendation by the previous reviewer

File: style.css (hunk 2 above)
Comment (rloterh) [reply style.css:115]: Well addressed

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi Sly-Cloud and Olawale-O! :wave: :wave:

####  Your project is complete! Nice job done for implementing the changes requested by the previous reviewer(s) :clap:  :clap: Your mobile menu is working correctly and your portfolio looks great! Congratulations 🎉🍾🍾 . There are however a few recommendations to implement to improve your project. Kindly check the optional comment below : 👍

## Good points 👍
No Linters errors ✔️
Semantics and syntax  are well used✔️

#### You may merge, good luck on your next project! :rocket:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- [OPTIONAL] Kindly check the comments above


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**